applications/nrf_desktop/README.rst

```diff
@@ -8,15 +8,25 @@ Nordic Semiconductor HID reference design.
 Overview
 ********
 
-This application is a reference design of a HID device.
-
 The code is currently work-in-progress and as such is not fully functional, verified, or
 supported for product development.
 
+Welcome to the firmware documentation for nRF52 Desktop, a reference design of
+a HID device.
+
+Depending on a configuration this application can function as mouse,
+gaming mouse or keyboard connected to host via BLE and/or USB.
+
+Project supports common input HW interfaces i.e. motion sensors, rotation
```

Comment:
"i.e." meaning "that is"? If so, this is a finished list, so let's transform it into a bullet point list.

Reply:
it should actually be e.g. or better "like"